inc/Engine/Admin/Settings/Page.php

```diff
@@ -639,6 +639,20 @@ private function assets_section() {
 			]
 		);
 
+		$delay_js_list_helper = sprintf(
+		// translators: %1$s = exclusion list, %2$s = opening </a> tag, %3$s = closing </a> tag.
+			__( 'If you have problems after activating this option, copy and paste the default exclusions to quickly resolve issues:<br><pre><code>%1$s</code></pre><br>Also, please check our %2$sdocumentation%3$s for a list of compatibility exclusions.', 'rocket' ),
```

Comment:
The html tags should not be part of the translatable string, it can be part of the %1$s placeholder instead